### PR TITLE
add bootstrap-material-datetimepicker v2.5.3 with git auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/T00rk/bootstrap-material-datetimepicker
**Watch 28
  Star 283
 Fork 124**
@PeterDaveHello  could u help me check this PR for #7546 ? thank you!
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update) https://github.com/T00rk/bootstrap-material-datetimepicker/issues/94
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Commit message in the first line is too long.

File: ajax/libs/bootstrap-material-datetimepicker/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  },
+  "author": "T00rk",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/T00rk/bootstrap-material-datetimepicker.git",
+    "basePath": "",
+    "files": [
+      "+(js|css)/bootstrap-material-datetimepicker*.+(css|js|map)",
+      "font/Material-Design-Icons*.+(eot|svg|ttf|woff|woff2)"
```
Comment (PeterDaveHello) [inline ajax/libs/bootstrap-material-datetimepicker/package.json]: fonts didn't been referenced in the js and css files, and we have: https://cdnjs.com/libraries/material-design-icons, so no need to add that.

**Comment by x09326** [comment]
Hi, I think you referenced the wrong number of the issue, So I modified it.

**Comment by maruilian11** [comment]
@PeterDaveHello updated.

**Comment by PeterDaveHello** [comment]
We started to minify the resources with map files for a while, so please also provide that, if you don't know how, reference to: https://github.com/PeterDaveHello/web-minify-helper, it'll help.

**Comment by maruilian11** [comment]
@PeterDaveHello sorry, i don't really understand your meaning, do you mean i didn't provide minified files?

**Comment by PeterDaveHello** [comment]
> @PeterDaveHello sorry, i don't really understand your meaning, do you mean i didn't provide minified files?

Did I say that? I say provide it with map file ...

**Comment by maruilian11** [comment]
@PeterDaveHello is that ok?

**Comment by maruilian11** [comment]
@PeterDaveHello update with minify-helper updated.

**Pull request merged**